argo-cd-release/action.yaml

```
@@ -57,6 +65,23 @@ runs:
       with:
         ssh-private-key: ${{ inputs.bot_ssh_private_key }}
 
+    - name: Authenticate to GCP
+      uses: google-github-actions/auth@v1.0.0
```

```suggestion
      uses: google-github-actions/auth@v1
```

Unless there's a reason to pin this to 1.0.0?